subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/TestResources.java

```diff
@@ -65,6 +65,16 @@ public void evaluate() throws Throwable {
         };
     }
 
+    // Remove brackets for spock iteration methods (e.g. this_is_a_test_method[0])
+    private String removeBracketsFromMethodName(String methodName) {
```

Comment:
based on @bmuschko's comment about replicating the Junit 4 tests for Junit 5, would this method be unnecessary?

Reply:
Now it's not necessary any more. I removed the spock extension completely and restored this code.